tsconfig.json

```diff
@@ -9,6 +9,7 @@
 		]
 	},
 	"include": [
-		"source"
+		"source",
+		"test"
```

Comment:
Adding another directory here means you need to update https://github.com/sindresorhus/is/blob/aeb3f74d653cf745abf2460f0896207de00c6059/package.json#L12 and https://github.com/sindresorhus/is/blob/aeb3f74d653cf745abf2460f0896207de00c6059/package.json#L22, and maybe more. As it will no longer compile `source` to the top-level of `dist`. It would be nice to keep `source` compiling to `dist` directly.

Reply:
Quite right -- only ran the local tests and didn't notice this change. Attempting to use the workaround from https://github.com/xojs/eslint-config-xo-typescript/issues/15; force-pushing an update.